File: StatsD_client.py
from datadog import initialize, statsd
import time
import random
#if __name__ == "__main__":
options = {
    'statsd_host':'host.docker.internal',
    'statsd_port':8125
}

initialize(**options)


while(1):
  statsd.distribution('custom.request.duration.avg', random.randint(0, 100), tags=["reason:statsd_test"])
  statsd.increment('custom.test_count.increment', random.randint(0, 100), tags=["reason:statsd_test"])
  statsd.gauge(`custom.test_gauge`, random.randint(0, 75), tags=["reason:statsd_test"])
  #statsd.decrement('gunicorn.test_count.decrement', tags=["environment:statsd_test"])
  time.sleep(2)

File: statsd_client.go
package main

import (
	"log"
	"time"

	"github.com/DataDog/datadog-go/v5/statsd"
)

func main() {
	statsd, err := statsd.New("127.0.0.1:8125")
	if err != nil {
		log.Fatal(err)
	}
	for {

		statsd.Count("jobs.enqueue", 1, []string{"environment:dev"}, 1)
		time.Sleep(10 * time.Second)
	}
}
